src/routes/plans/[id]/+page.ts

```diff
@@ -27,6 +27,7 @@ export const load: PageLoad = async ({ parent, params, url }) => {
       return {
         initialActivityTypes,
         initialPlan,
+        initialResourceTypes,
```

Comment:
We can remove this line since we are not going to use these from here

Reply:
Removed. Should I remove the other references to resource types in this file or do you want to just leave that to the other PR?

Reply:
Looks good! You can leave them. Will merge the other PR shortly and we can rebase.